data/init.sh

```diff
@@ -1,2 +1,10 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+VERSION=$(grep DOCKER_IMAGE_CBIOPORTAL ../.env | cut -d '=' -f 2-)
+
+# Get the schema
+docker run --rm -it $VERSION cat /cbioportal/db-scripts/src/main/resources/cgds.sql > cgds.sql
```

Comment:
Trying to get the docker compose scripts working with the 6.0.x release and this doesn't quite seem to work. When trying to run `docker compose up` the database container fails to boot with the error

```
cbioportal-database-container          | 2024-03-09 22:52:39+01:00 [Note] [Entrypoint]: /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/cgds.sql
cbioportal-database-container          | ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'cat: /cbioportal/db-scripts/src/main/resources/cgds.sql: No such file or directo' at line 1
cbioportal-database-container exited with code 1
```

So it seems like the literal `cat /path` statement is ending up in the `.sql` file. The database container reboots and then stays up, but the migration script in the main cbioportal-container subsequently fails because it tries to migrate based on tables that don't exist (since this initial schema load failed.)